src/modules/dashboard/components/ActionsPage/ActionsComponents/MintTokenMotion.tsx

```diff
@@ -4,7 +4,7 @@ import { FormattedMessage } from 'react-intl';
 
 import Numeral from '~core/Numeral';
 import ActionsPageFeed, {
-  ActionsPageFeedItem,
+  ActionsPageFeedItemWithIPFS,
   SystemMessage,
 } from '~dashboard/ActionsPageFeed';
 import { ColonyMotions, ColonyAndExtensionsEvents } from '~types/index';
```

Comment:
Something broken this page, as it will no longer render properly for any motion created through your new sagas _(including the original Mint Tokens Motion)_

![Screenshot from 2021-04-26 10-01-27](https://user-images.githubusercontent.com/1193222/116042000-9751bc00-a676-11eb-8439-1dc1b88f6d58.png)

It's most likely not from your changes to this file, but please investigate it and fix it.